hermeto/interface/cli.py

```diff
@@ -227,7 +227,9 @@ def fetch_deps(  # noqa: D103; docstring becomes part of --help message
         resolve_path=True,
         help="Write output files to this directory.",
     ),
-    dev_package_managers: bool = typer.Option(False, "--dev-package-managers", hidden=True),
+    dev_package_managers: bool = typer.Option(
+        False, "--dev-package-managers", hidden=True, help="DEPRECATED: No longer has any effect."
+    ),
```

Comment:
I understand the change, but it's a complete NOP, because of  `hidden=True` and so the option is not even visible on the CLI. Well, since the flag is invisible in help, but documented publicly in our docs I'd say you'll need a new commit that formally exposes the flag on the CLI to adhere to the docs (probably the first commit in this series), only to deprecate it this way a few commits later :) .